extensions/ql-vscode/src/interface.ts

```diff
@@ -488,13 +489,14 @@ export class InterfaceManager extends DisposableObject {
       schema.name,
       {
         offset: schema.pagination?.offsets[pageNumber],
-        pageSize: RAW_RESULTS_PAGE_SIZE
+        pageSize: PAGE_SIZE.getValue()
```

Comment:
Nit: you can fetch the page size setting once and save it to a variable for the below uses.

Reply:
done